src/css/_base.scss

```diff
@@ -181,7 +205,7 @@ code {
 
 pre {
     padding: 8px 12px;
-    overflow-x: scroll;
+    overflow-x: auto;
```

Comment:
What is this about?

Reply:
There is an always-visible scrollbar in code content even when the window is large enough: https://letsencrypt.org/docs/revoking/ 
![image](https://user-images.githubusercontent.com/1955774/68775390-af727e80-062e-11ea-87bc-493f4d5b62a8.png)
That scrollbar is useless, and it became worse in dark mode:
![image](https://user-images.githubusercontent.com/1955774/68775672-2b6cc680-062f-11ea-849a-ebfc953807ba.png)